src/client/app/containers/UIOptionsContainer.js

```diff
@@ -23,7 +23,10 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
 	return {
 		selectMeters: newSelectedMeterIDs => dispatch(changeSelectedMeters(newSelectedMeterIDs)),
-		fetchMetersDataIfNeeded: () => dispatch(fetchMetersDataIfNeeded())
```

Comment:
I think `fetchMetersDetails` is a lot clearer than `fetchMetersData`, which can be confused with the data we get from meters (as in readings).